lib/purchase.js

```diff
@@ -0,0 +1,26 @@
+const { Model, DataTypes } = require("sequelize");
+const sequelize = require("../config/connection");
+
+class Purchase extends Model {}
+
+Purchase.init(
+  {
+    price: {
+      type: DataTypes.INTEGER,
+      allowsNull: false,
+    },
+    type: {
+      type: DataTypes.STRING,
+      allowsNull: false,
+    },
+  },
```

Comment:
i think type should be referencing a category table so that the user can choose from different options